scopes/workspace/workspace/workspace.docs.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-description: A local workspace for component development and composition
-labels: ['core aspect']
+description: A component development workspace designed for the construction of distributed systems with a monolith-like dev experience.
```

Comment:
A development workspace for distributed, component-driven software.

Reply:
done